### ISSUE TITLE
Bug when layer is empty

### ISSUE THREAD

**Issue by eurochriskelly** [johannesboyne/node-dxf#1]
Hi Johannes,
Thanks for this module. It has come in handy for a project I am working on. I found a bug (line 65) when importing empty layer which I fixed as follows.

```
if (polygons[polygons.length-1].points.length) {
    polygons[polygons.length-1].points[polygons[polygons.length-1].points.length-1].y = Number(chunk.toString());
}
```

Regards,
Chris

**Comment by johannesboyne** [comment]
Hi Chris,

Thanks, great to hear this module is being used!
May I ask you to open a pull-request with the fix - would love to merge it into the project and update the NPM package?

Thanks & regards,

Johannes

**Comment by johannesboyne** [comment]
Thanks @eurochriskelly, I've merged the PR and updated the NPM package (v0.0.3)!

**Comment by eurochriskelly** [comment]
Great! Luckily I remembered to update my package.json!

On Mon, Aug 25, 2014 at 7:59 PM, Johannes Boyne notifications@github.com
wrote:

> Thanks @eurochriskelly https://github.com/eurochriskelly, I've merged
> the PR and updated the NPM package (v0.0.3)!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/johannesboyne/node-dxf/issues/1#issuecomment-53301437
> .

## 

/Chris Kelly
